src/ProtocolZoo/ProtocolZoo.jl

```diff
@@ -131,7 +132,8 @@ end
         tag!(a, EntanglementCounterpart, prot.nodeB, b.idx)
         # tag local node b with EntanglementCounterpart remote_node_idx_a remote_slot_idx_a
         tag!(b, EntanglementCounterpart, prot.nodeA, a.idx)
-
+        
+        @debug "EntanglerProt between $(prot.nodeA) and $(prot.nodeB): Entangled .$(a.idx) and .$(b.idx)"
```

Comment:
Please let me know if these new debug statements seem excessive, I'll remove them : )

Reply:
nah, we should not shy away from adding a ton of debug info